client/src/app/site/motions/models/view-motion-poll.ts

```diff
@@ -18,17 +18,16 @@ export class ViewMotionPoll extends BaseProjectableViewModel<MotionPoll> impleme
     }
 
     public getSlide(): ProjectorElementBuildDeskriptor {
-        /*return {
```

Comment:
The projector is still not implemented on server side. This will not work. Please revert and raise a `TODO` *or* (if this function is called implecit) return a dummy element and mark, that it is still TODO.